tests/test_adjoint.py

```diff
@@ -27,6 +29,8 @@ class TestAdjoint(object):
         ('layers', (60, 70), 'OT2', 8, 2, acoustic_setup),
         ('layers', (60, 70), 'OT2', 4, 2, acoustic_setup),
         ('layers', (60, 70), 'OT4', 2, 2, acoustic_setup),
+        # 2D test with 2 layers and freesurface
+        ('layers-fs', (200, 80), 'OT2', 4, 2, acoustic_setup),
```

Comment:
We trying to avoid these "larger" sizes for CI as it increases the test time quite a bit, would prefer the standard sizes used with the other cases and keep the `tn` as it is

Reply:
you're right , I'll keep the smaller sizes and 500. ms